scripts/publish_zap_compatible.py

```diff
@@ -35,8 +35,8 @@ def scan_and_remove(obj, mappings):
             (
                 ("format", "date"),
                 ("personalisation", None),
-                ("/callbacks/message-status", None),
-                ("/<client-provided-URI>", None),
+                ("/<client-provided-message-status-URI>", None)
+                ("/<client-provided-channel-status-URI>", None)
```

Comment:
This is making the zap scan fail:
```
scripts/publish_zap_compatible.py:38: SyntaxWarning: 'tuple' object is not callable; perhaps you missed a comma?
  ("/<client-provided-message-status-URI>", None)
Traceback (most recent call last):
  File "scripts/publish_zap_compatible.py", line 38, in <module>
    ("/<client-provided-message-status-URI>", None)
TypeError: 'tuple' object is not callable
make: *** [Makefile:133: zap-security-scan] Error 1

##[error]Bash exited with code '2'.
```
https://dev.azure.com/NHSD-APIM/API%20Platform/_build/results?buildId=236521&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=9b7ee503-4929-52e5-9440-042a18698d3f&l=176

```suggestion
                ("/<client-provided-message-status-URI>", None),
                ("/<client-provided-channel-status-URI>", None),
```